classes/class-wc-connect-nux.php

```diff
@@ -93,6 +93,14 @@ public function show_pointers( $hook ) {
 			wp_enqueue_script( 'wc_services_admin_pointers' );
 		}
 
+		public function get_dismissed_pointers() {
+			$data = get_user_meta( get_current_user_id(), 'dismissed_wp_pointers', true );
+			if ( is_string( $data ) && 0 < strlen( $data ) ) {
```

Comment:
The problem is that when we later add a new element to the imploded array, we'll get something like this:
```
implode(array('', 'new_dismissed_pointer')); // ',new_dismissed_pointer'
```